android/src/main/java/com/tailscale/ipn/IPNService.kt

```diff
@@ -41,6 +41,7 @@ open class IPNService : VpnService(), libtailscale.IPNService {
           // This means we were started by Android due to Always On VPN.
           // Get the application to make sure it's been initialized, then
           // request the VPN.
+          showForegroundNotification()
```

Comment:
Have you tested this? I think it might not work because I'm not sure that `startForeground()` has been called in this case.

Reply:
Using non-foreground notification. Also canceling notifications when App closes to clear any lingering notifications, and only showing a disconnected notification when in the Stopped state.